### PR TITLE
feat: [ZNO-2042] add analytical DB (clickhouse)

### PR DESCRIPTION
**Pull request by dkaliberda**
### SUMMARY
Added analytical database  (clickhouse)

### TESTING INSTRUCTIONS
- open path /databaseview/list/
- you should see the "clickhouse" database
- the database connection test must be successful

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docker-compose-non-dev.yml [hunk 1]
```diff
@@ -92,10 +92,25 @@ services:
     healthcheck:
       disable: true
 
+  clickhouse:
```
Comment (NikolayBorovenskiy) [inline docker-compose-non-dev.yml:95]: I suggest adding `superset-` as a prefix to follow the style to the file.

File: docker-compose-non-dev.yml [hunk 2]
```diff
@@ -92,10 +92,25 @@ services:
...
+    image: clickhouse/clickhouse-server:22.3
+    user: "101:101"
+    env_file: docker/.env-non-dev
+    container_name: clickhouse
```
Comment (NikolayBorovenskiy) [inline docker-compose-non-dev.yml:99]: And here please add a prefix `superset`.

File: docker-compose.yml [hunk 3]
```diff
@@ -157,10 +157,25 @@ services:
...
+    user: "101:101"
+    env_file: docker/.env
+    container_name: clickhouse
+    hostname: clickhouse
+    restart: unless-stopped
+    volumes:
+      - clickhouse:/var/lib/clickhouse
+    ports:
+      - "127.0.0.1:8123:8123"
+      - "127.0.0.1:9000:9000"
```
Comment (NikolayBorovenskiy) [inline docker-compose.yml:171]: and here

File: docker/.env [hunk 4]
```diff
@@ -51,3 +51,10 @@ SUPERSET_LOAD_EXAMPLES=yes
 CYPRESS_CONFIG=false
 SUPERSET_PORT=8088
 MAPBOX_API_KEY=''
+
+SUPERSET_CLICKHOUSE_INIT=yes
+CLICKHOUSE_URI=clickhousedb://superset:superset@clickhouse:8123/superset
+CLICKHOUSE_DB=superset
+CLICKHOUSE_USER=superset
+CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT=1
+CLICKHOUSE_PASSWORD=superset
```
Comment (NikolayBorovenskiy) [inline docker/.env:60]: add a trailing new line.

**Review by NikolayBorovenskiy — CHANGES_REQUESTED** [review]

File: docker-compose-non-dev.yml (hunk 1 above)
Comment (dkaliberda) [reply docker-compose-non-dev.yml:95]: I do not agree here, because additional services do not have a prefix.

File: docker/.env (hunk 4 above)
Comment (dkaliberda) [reply docker/.env]: fixed

File: docker-compose-non-dev.yml (hunk 2 above)
Comment (dkaliberda) [reply docker-compose-non-dev.yml]: fixed

File: docker-compose.yml (hunk 3 above)
Comment (dkaliberda) [reply docker-compose.yml]: fixed

**Review by NikolayBorovenskiy — APPROVED** [review]

**Pull request merged**